page-components/form-fields.md

```diff
@@ -672,6 +672,58 @@ To optimize screen reader accessibility, lay out checkboxes vertically rather th
 {:/nomarkdown}
 
 </div>
+<br>
+### Nested checkboxes
```

Comment:
@cfarm - I thought this was already merged. It's already on the live page. Do you know why it's showing up here?